lib/Common/DataStructures/FixedBitVector.h

```diff
@@ -242,18 +242,19 @@ void BVFixed::SetRange(Container* value, BVIndex start, BVIndex len)
     BVIndex oStart = BVUnit::Offset(start);
     BVIndex oEnd = BVUnit::Offset(end);
 
-    BVUnit::BVUnitTContainer temp;
-    BVUnit::BVUnitTContainer* bits;
+    typedef typename std::conditional<sizeof(Container) == 1,uint8_t,BVUnit::BVUnitTContainer>::type operandtype;
```

Comment:
Note: I'm aware that this change could have a performance penalty if it goes the wrong way; I'm looking at it.